### PR TITLE
Start working on cp integration tests

### PR DESCRIPTION
**Pull request by thieman**
WIP, need to refactor the Git stuff a bit so we stop talking to GitHub as part of integration tests

### CODE REVIEW

**Comment by thieman** [comment]
@paetling REVIEW ME WOOOOO

Couple things going on in this PR:

1. Fix a bad assumption on where user's home dirs live (bug fix)
1. Accept SSH_AUTH_SOCK if it's already set for the daemon process. This lets integration tests pass the socket in when running in a CI environment
1. Set up a tiny local Git repo for CI clones so we don't have to talk to GitHub
1. Write one cp test to display some of the new functionality

File: tests/fixtures.py [hunk 1]
```diff
@@ -36,7 +36,12 @@ def basic_specs_fixture():
...
+    /tmp/fake-repo should be set up before using this fixture."""
     for bundle in range(num_bundles):
-        name = 'busybox-{}'.format(_num_to_alpha(bundle))
+        name = 'busybox{}'.format(_num_to_alpha(bundle))
         _write('bundle', name, {'description': 'Busybox bundle', 'apps': [name]})
-        _write('app', name, {'repo': 'github.com/fake/repo', 'image': 'busybox'})
+        _write('app', name, {'repo': '/tmp/fake-repo',
+                             'mount': '/repo',
+                             'image': 'busybox',
+                             'commands': {'always': 'sleep 999999999'}})
```
Comment (paetling) [inline tests/fixtures.py]: why this long but not infinitey?

Comment (thieman) [inline tests/fixtures.py]: busybox doesn't support infinite

**Comment by paetling** [comment]
:ghost: :cow:

**Pull request merged**